src/instruments/src/EFB/Settings/Pages/RealismPage.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { } from 'react';
```

Comment:
Remove the dangling brackets
```suggestion
import React from 'react';
```